package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "chromedriver": "^2.31.0",
     "glob": "^7.1.1",
+    "jimp": "^0.2.28",
```

Comment:
this project is nice because it doesn't require any external dependencies (e.g. imagemagick). Downside is it might be a bit slower though I'm not sure on that.

Reply:
Do you have any other information regarding `jimp`?

Reply:
https://github.com/oliver-moran/jimp

Reply:
Any information on choosing this package over another?

Reply:
It has crop and compare with no external dependencies